learning_resources/migrations/0079_ocw_pages_trailing_slash.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.16 on 2024-12-06 22:02
+
+from django.db import migrations
+
+
+def ensure_trailing_slashes(apps, schema_editor):
+    LearningResource = apps.get_model("learning_resources", "LearningResource")
+
+    for resource in LearningResource.objects.filter(
+        resource_type="course", offered_by_id="ocw"
```

Comment:
Could also filter by `etl_source`  alone, I think the end result will be the same:
```python
for resource in LearningResource.objects.filter(etl_source="ocw"):
```